### DISCUSSION TITLE
AWS error

### DISCUSSION THREAD

**Discussion by dmganly** [Budibase/budibase#6267] [Help] [answered]
Hi Guys,

I'm new to most of this stuff and learning curve is pretty steep when you add up Linux on Windows, Github, budibase interface, possibilities of deployment etc..... I have tried to follow this excellent video 

but receiving error because seemingly I'm not well setup with roles, permissions on AWS 
Here's the error I receive when I try the command......

eksctl create cluster --name budibase-cluster --region eu-west-2 

Error: checking AWS STS access – cannot get role ARN for current session: operation error STS: GetCallerIdentity, failed to sign request: failed to retrieve credentials: failed to refresh cached credentials, no EC2 IMDS role found, operation error ec2imds: GetMetadata, request send failed, Get "http://169.254.169.254/latest/meta-data/iam/security-credentials/": dial tcp 169.254.169.254:80: i/o timeout

It would be super handy if there was any easy to follow run through for the prerequisites of setting up budibase on AWS rather than me having to read through the complete docs of AWS.....  Many thanks

**Comment by dmganly** [comment]
This excellent video - [https://docs.budibase.com/docs/kubernetes-k8s ](url)

**Comment by shogunpurple** [comment, marked as answer]
@dmganly this issue has nothing to do with budibase. This is an AWS issue with `eksctl` and a configuration issue on your side, or there is a genuine bug in `eksctl` https://github.com/weaveworks/eksctl/issues/321 (?).

Do you have your AWS profiles set up correctly?  When you run a `aws configure list` what is set?

**Comment by dmganly** [comment]
Some progress but now getting

aws configure list....

      Name                    Value             Type    Location
      ----                    -----             ----    --------
   profile                <not set>             None    None
access_key     ****************WJGY shared-credentials-file
secret_key     ****************LpZ/ shared-credentials-file
    region                eu-west-2      config-file    ~/.aws/config
